browser/sync_page_mapping.go

```diff
@@ -121,8 +122,13 @@ func syncMapPage(vu moduleVU, p *common.Page) mapping { //nolint:gocognit,cyclop
 				_, err := handler(sobek.Undefined(), vu.Runtime().ToValue(mapping))
 				return err
 			}
-			runInTaskQueue := func(m *common.ConsoleMessage) {
+			runInTaskQueue := func(a any) {
```

Comment:
It's nice that we generalize it this way 👍 But, also consider if we can use Go generics here.

Reply:
I've linked this comment to a [comment](https://github.com/grafana/xk6-browser/issues/1227#issuecomment-2393348036) in the generalize `page.on` events [issue](https://github.com/grafana/xk6-browser/issues/1227).

We can tackle the generalization after merging in this PR 👍 

I'll resolve this comment so that I can work on the other feedback.